x-pack/plugin/sql/src/test/java/org/elasticsearch/xpack/sql/analysis/analyzer/VerifierErrorMessagesTests.java

```diff
@@ -1257,4 +1258,62 @@ public void testBinaryFieldWithNoDocValues() {
         assertEquals("1:34: Binary field [binary] cannot be used for ordering unless it has the doc_values setting enabled",
             error("SELECT binary FROM test ORDER BY binary"));
     }
+
+    public void testDistinctIsNotSupported() {
+        assertEquals("1:8: SELECT DISTINCT is not yet supported", error("SELECT DISTINCT int FROM test"));
+    }
+
+    public void testExistsIsNotSupported() {
+        assertEquals("1:33: EXISTS is not yet supported", error("SELECT test.int FROM test WHERE EXISTS (SELECT 1)"));
+    }
```

Comment:
These are tested in the REST tests too. Not sure why there and not here, but wondering if it'd be worth removing them from there, dunno if it's worth having them in both places.

Reply:
+1 Everything that can be covered by unit tests should be removed from integration tests, which already take long time to execute.

Reply:
Done.